docs/FAQ/FAQ.qmd

```diff
@@ -27,7 +27,32 @@ eatRep::pool.R2(r2)
 
 ## 2. DIF-Richtung in Conquest/eatModel: Was bedeuten negative Werte in der "estDif"-Spalte?
```

Comment:
quite nice that you're editing my chunks (don't miunderstand, I really like what you added, in sum), but what about adding "your" chunks (see t:\SIG\SIG Methoden\Liste methodischer Fragen.docx) ?